src/main/java/mate/academy/config/MapperConfig.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.config;
+
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.NullValueCheckStrategy;
+
+@org.mapstruct.MapperConfig(
+        componentModel = "spring",
+        injectionStrategy = InjectionStrategy.CONSTRUCTOR,
+        nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS,
+        implementationPackage = "<PACKAGE_NAME>.impl"
```

Comment:
The placeholder '<PACKAGE_NAME>' in 'implementationPackage' should be replaced with the actual package name where the mapper implementations should be generated. This is necessary for the configuration to work correctly.